scripts/rnnlm/lmrescore_nbest.sh

```diff
@@ -180,17 +183,26 @@ if [ $stage -le 6 ]; then
     rnnlm/compute_sentence_scores.sh $rnndir $adir.JOB/temp \
                                    $adir.JOB/words_text $adir.JOB/lmwt.rnn 
 fi
+
 if [ $stage -le 7 ]; then
+  echo "$0: doing average on forward and backward scores."
```

Comment:
The same issue here-- you are changing this script in a very non-back-compatible way.  I don't see what this has to do with the Chime challenge anyway, I think it would be better if you leave this to people like @hainan-xv who are working primarily on RNNLMs.